upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -98,6 +100,9 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["EncodeGCELabel"] = gce.EncodeGCELabel
 
 	dest["DnsControllerImage"] = tf.DnsControllerImage
+
+	// TODO: Only for AWS?
+	dest["AWSRegion"] = tf.AWSRegion
```

Comment:
I'm just following other template functions which seems to be specific to other cloud providers like GCE. However, I felt the need to leave a TODO comment, because it felt wrong to do so. Like the others, it's up to the template to have an if statement check to see whether or not the .CloudProvider is set to aws, before calling this function. Otherwise, it'll just error out.

I'll see if it is trivial to move that to apply_cluster.go.